snapcraft/main.py

```diff
@@ -361,7 +368,11 @@ def _run_store_command(args):  # noqa: C901
     elif args['status']:
         snapcraft.status(
             args['<snap-name>'], args['--series'], args['--arch'])
-    elif args['revisions']:
+    elif args['history']:
+        deprecations.handle_deprecation_notice('history')
+        snapcraft.revisions(
+            args['<snap-name>'], args['--series'], args['--arch'])
```

Comment:
Rather than calling `snapcraft.revisions` twice, I suggest something a little less performant but less error-prone:

```python
elif args['revisions'] or args['list-revisions'] or args['history']:
    if args['history']:
        deprecations.handle_deprecation_notice('dn4')
    snapcraft.revisions([...])
```